gugel-project-2/App.js

```diff
@@ -0,0 +1,90 @@
+// In App.js in a new project
+import * as React from 'react';
+import { View, FlatList, ScrollView, Text, Button, SectionList, StyleSheet, TextInput, TouchableWithoutFeedback } from 'react-native';
+import {List, ListItem, SearchBar } from "react-native-elements";
+import { NavigationContainer, useNavigation, withNavigation } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+
+let movieList = [{title: 'Movie A', date: 1997, key: '1'}, {title: 'Movie B', date: 2000, key: '2'}, {title: 'Movie C', date: 1942, key:'3'}]
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 48
+  },
+})  
+let ogMovieList = movieList
```

Comment:
Since we never actually modify our list(the `.filter` method returns a new array and isn't destructive), we don't need this variable at all. We can just use `movieList` in the rest of our app